package.json

```diff
@@ -61,7 +62,7 @@
   "devDependencies": {
     "autoprefixer": "^10.2.5",
     "cypress": "^6.6.0",
-    "node-sass": "^5.0.0",
+    "node-sass": "^6.0.1",

```

Comment:
Could you revert this change? I'm currently doing some project refactoring that also involves removing the use of `node-sass`